docs/api/math/Triangle.html

```diff
@@ -79,7 +79,9 @@ <h3>[method:Boolean containsPoint]( [page:Vector3 point] )</h3>
 		<div>
 		[page:Vector3 point] - [page:Vector3] to check.<br /><br />
 
-		Returns true if the passed vector is within the triangle.
+		Returns true if the passed vector is within the triangle in
+		[link:https://en.wikipedia.org/wiki/Barycentric_coordinate_system barycentric coordinates]
+		(the vector is projected onto the plane of the triangle before checking containment).
```

Comment:
Maybe
```
Returns true if the passed point, when projected onto the plane of the triangle, lies within the triangle.
```

Reply:
Updated, with small rearrangement:

```
Returns true if the passed point lies within the triangle, when projected onto the plane of the triangle.
```

Reply:
Never mind, going with your suggested wording.

Reply:
Hehe. I had trouble wording that one, too. :)